lib/cli/cli.js

```diff
@@ -983,6 +988,15 @@ module.exports.parseCommandLine = function parseCommandLine() {
         );
       }
     })
+    .coerce('crawler', crawler => {
+      if (crawler.exclude) {
+        if (!Array.isArray(crawler.exclude)) {
+          crawler.exclude = [crawler.exclude];
+        }
+        crawler.exclude = crawler.exclude.map(e => new RegExp(e));
```

Comment:
Wasn't sure where to put this. At least here it would fail early in case the user provided an invalid regular expression.